data_api/main.tf

```diff
@@ -0,0 +1,11 @@
+module "elasticdump" {
+  source        = "git::https://github.com/wellcometrust/terraform.git//ecs_script_task?ref=v1.0.0"
+  task_name     = "elasticdump"
+  app_uri       = "wellcome/elasticdump:latest"
+  task_role_arn = "${module.ecs_elasticdump_iam.task_role_arn}"
+
+  env_vars = [
+    "{\"name\": \"BUCKET\", \"value\": \"${var.elasticdump_config_bucket}\"}",
```

Comment:
Ick. We should make the module not be this way!

Reply:
I'm not sure it needs to at a more current version. 

This is really just a placeholder to add _some script task_ in the future.